### DISCUSSION TITLE
[FEAT] Add a section where we can see what boost is applied on our farm

### DISCUSSION THREAD

**Discussion by 0xbhagi** [sunflower-land/sunflower-land#4024] [Ideas]
Summary of Request
As a player of Sunflower Land, I would like to have a dedicated section in the game where I can view the active boosts applied to my farm, in order to better manage my resources and strategize my gameplay.

**Comment by 0xBrink** [comment]
I'd love to see this implemented too. A feature where you can hover over the items/SFT's would be great too
However, a section with the boosts listed would be preferred from my side. 
It would be nice if the boosts that interfere with each other like (fx Lunar Table and Scarecrow) would be added up together, so it's clear how much extra yield I get from those 2 boosts
